appconfiguration/feature-management-sample/README.md

```diff
@@ -15,42 +15,7 @@ Start the application and check the resulting console output to check the return
 $ mvn spring-boot:run
 ```
 
-2. Check the returned value. The feature `Beta` has one filter `Random` which defines
-
-### How to run with Azure Configuration Service
-
-#### Prepare data
-
-1. Create a Configuration Store if not exist.
-
-2. Import the data file src/main/resources/data/sample-data.json into the Configuration Store created above. Under `For language` select `Other`. Under `File type` select `Yaml`.
-
-#### Configure the bootstrap.yaml
-
-Change the connection-string value with the Access Key value of the Configuration Store created above.
-
-#### Run the application
-Start the application and access http://localhost:8080 to check the returned value. Different commands for different scenarios are listed below.
-
-1. Load properties similar with from application.properties, i.e., keys starting with /application/
-```
-$ mvn spring-boot:run
-```
-
-2. Load properties similar with from application_dev.properties, i.e., keys starting with /application_dev
-```
-$ mvn -Dspring.profiles.active=dev spring-boot:run
-```
-
-3. Load properties similar with from foo.properties, i.e., keys starting with /foo/
-```
-$ mvn -Dspring.application.name=foo spring-boot:run
-```
-
-4. Load properties similar with from foo_dev.properties, i.e., keys starting with /foo_dev/
-```
-$ mvn -Dspring.application.name=foo -Dspring.profiles.active=dev spring-boot:run
-```
+2. Check the returned value. The feature `Beta` has one filter `Random` which defines. You should see the following information displayed: **RUNNING : application** or **RUNNING : beta**.
```

Comment:
which defines the `Beta` feature will be activated randomly with some `chance` value.